### PR TITLE
Fix IE11 bugs with search

### PR DESCRIPTION
**Pull request by Badlapje**
Prior to this change there were two problems in IE11:
- the search reset button did not appear
- the search/sort did not work

This change repairs the search & sort, as well as ensuring the reset button works as expected.  

[Issue brought up as part of the feedback round](https://wiki.surfnet.nl/display/coininfra/Bevindingen+nav+tests+door+supportteam)

### CODE REVIEW

File: src/OpenConext/EngineBlockFunctionalTestingBundle/Features/Context/EngineBlockContext.php [hunk 1]
```diff
@@ -390,7 +390,7 @@ public function iDoNotSeeACertainFormFieldOnTheProcessForm($formFieldName)
      */
     public function iSeeTheRequestAccessButton()
     {
-        $selector = 'article.wayf__idp--noAccess';
+        $selector = '.wayf__idp--noAccess';
```
Comment (pablothedude) [inline src/OpenConext/EngineBlockFunctionalTestingBundle/Features/Context/EngineBlockContext.php:393]: Removal of element in selector :100:

File: theme/base/stylesheets/pages/wayf/remainingIdps.scss [hunk 2]
```diff
@@ -31,7 +31,7 @@
             display: block;
             font-size: $f-large;
             height: 3rem;
-            line-height: 1.5;
+            line-height: 3rem;
```
Comment (pablothedude) [inline theme/base/stylesheets/pages/wayf/remainingIdps.scss:34]: Usage of rem :100:

File: theme/package.json [hunk 3]
```diff
@@ -22,7 +22,9 @@
     "buildtheme": "npm run build:clean && npm run build:js && npm run build:css && npm run build:copy",
     "build:clean": "rimraf ../web/javascripts/* ../web/stylesheets/*",
     "build:js": "mkdir -p ../web/javascripts && npm run build:js:babelify",
+    "build:js:debug": "mkdir -p ../web/javascripts && npm run build:js:babelify:debug",
```
Comment (pablothedude) [inline theme/package.json:25]: I think `mkdir -p ../web/javascripts ` could be omitted.

**Review by pablothedude — APPROVED** [review]

File: theme/package.json (hunk 3 above)
Comment (Badlapje) [reply theme/package.json:25]: Unless it's the first time you run a js build script.  So included it for robustness

**Pull request merged**